README.md

```diff
@@ -84,9 +87,27 @@ export default {
   // ...
   // Your loader options as svgLoader object
   svgLoader: {
-    svgo: {
+    svgoConfig: {
       plugins: [
-        { prefixIds: true },
+        { prefixIds: false }, // Disables prefixing for SVG IDs
```

Comment:
nit: remove the trailing comma after `}`

Reply:
```suggestion
        { prefixIds: false } // Disables prefixing for SVG IDs
```

Reply:
Right 😋